commands/rm.go

```diff
@@ -100,3 +127,56 @@ func rm(ctx context.Context, dockerCli command.Cli, ng *store.NodeGroup, keepSta
 	}
 	return err
 }
+
+func rmAllInactive(ctx context.Context, txn *store.Txn, dockerCli command.Cli, in rmOptions) error {
+	ctx, cancel := context.WithTimeout(ctx, 20*time.Second)
+	defer cancel()
+
+	ll, err := txn.List()
+	if err != nil {
+		return err
+	}
+
+	builders := make([]*nginfo, len(ll))
+	for i, ng := range ll {
+		builders[i] = &nginfo{ng: ng}
+	}
+
+	eg, _ := errgroup.WithContext(ctx)
+	for _, b := range builders {
+		func(b *nginfo) {
+			eg.Go(func() error {
+				if err := loadNodeGroupData(ctx, dockerCli, b); err != nil {
+					return errors.Wrapf(err, "cannot load %s", b.ng.Name)
+				}
+				if b.ng.Dynamic {
+					return nil
+				}
+				var kn []store.Node
+				for idx, n := range b.ng.Nodes {
+					d := b.drivers[idx]
+					if d.info == nil || d.info.Status != driver.Running {
+						_ = d.di.Driver.Rm(ctx, true, !in.keepState, !in.keepDaemon)
```

Comment:
Could we still show these errors? They should not fail the removal of the builder but for the user these would probably be the most useful.

Reply:
sure sgtm